lib/shopify_import_massager.ex

```diff
@@ -136,13 +227,46 @@ defmodule ShopifyImportMassager do
     File.close(output_file)
   end
 
-  def substitute_input_line(input_line, substitution_map) do
-    substitution_map
-    |> Enum.reduce(
-      input_line,
-      fn %{order_name: order_name, replace_order_number: replace_order_number}, adjusted_line ->
-        String.replace(adjusted_line, order_name, replace_order_number, global: true)
-      end
-    )
+  def get_column_index(file_type) do
+    # If it's an orders file, then only substitute on the first column.
+    # If it's a transaction, then only substitute on the second column.
+    # If it's a return, then only substitute on the first column.
+    # Remember: 0-based indexing
+    case file_type do
+      :orders -> 0
+      :transactions -> 1
+      :returns -> 0
+    end
+  end
+
+  # TODO It is slow and inefficient to do a line-by-line substitution for each
```

Comment:
Some caveats here. Still not sure if this is the best approach, but it does work a lot faster.